201-web-app-asev2-create/azuredeploy.parameters.json

```diff
@@ -6,7 +6,7 @@
             "value": "GEN_UNIQUE"
         },
         "aseLocation": {
-            "value": "South Central US"
+            "value": "changeme"

```

Comment:
see the readme for how to use samples with existing resources

Reply:
Corrected